### ISSUE TITLE
[SF][0.131.20][UploadCollection][BUG] a11y issue: Instruction of actionable elements is supposed to be provided

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#4344]
**Describe the bug**
Once the focus is on the attachment list item, the screen reader announces all the content from the list item. But, it is necessary for a screen reader user to have the instruction provided saying "Contains actionable elements, press tab to access them".

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-uploadcollection--with-upload-collection-item
2. Focus on UploadCollection item
3. See NVDA Speech Viewer

**Expected behavior**
It is necessary for a screen reader user to have the instruction provided saying "Contains actionable elements, press tab to access them".

**Screenshots**
![uploadCollection](https://user-images.githubusercontent.com/30744017/141937592-2a6e847c-a6b2-412d-aa75-9a2b56668169.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 0.31.19
`@ui5/webcomponents-react` version: 0.131.20
Operating System: macOS Big Sur
Browser: 

**Additional context**
Here is the original JIRA ticket: https://jira.successfactors.com/browse/WEF-17098

**Comment by IlianaB** [comment]
Hello @Lukas742 ,

In order to process the issues correctly, we would like to have the following information provided:
- Reference to the suspected violated accessibility requirement (e.g. Web Content Accessibility Guidelines, WCAG 2.0, BITV 2.0, EN 301 549)

Regards,
Iliana

**Comment by Lukas742** [comment]
Hi @IlianaB 

I did not raise this issue, I just forwarded it ;)

@i323808 could you please provide the information required by @IlianaB.

**Comment by i323808** [comment]
Hi @IlianaB ,

Below are the details provided by JIRA ticket reporter(Vinayak Satpute):

WCAG/BITV equivalent:
WCAG 2.0, 3.3.2, Level A, Labels or instructions
WCAG 2.0, 2.4.6, Level AA, Headings and Labels
BITV 2.0, 2.4.6, Prio 1, Beschreibungen
BITV 2.0, 3.3.2, Prio 1, Beschriftungen

Best Regards,
Jane

**Comment by gmkv** [comment]
Hello @i323808 
The UploadCollection is a container for other elements.
Accessibility-wise, it is implemented as a list, and it gets announced correctly as such.

Where does the neccessity come from for a container to provide the literal text "Contains actionable elements, press tab to access them" to the user?
Do you have knowledge of other container elements, such as the <ui5-list>, or OpenUI5's List or UploadCollection having this announcement?

I read the concerns you provided and do not see anywhere in the sentences of the standards and did not find violations. Where is the exact text in the standards you believe that we have violated? 

Regards,
Georgi

**Comment by gmkv** [comment]
Hi @i323808 

I'm closing this issue due to inactivity.
Feel free to reply and we can reopen it again!

Regards,
Georgi
